language/stdlib/modules/libra_system_2.move

```diff
@@ -0,0 +1,264 @@
+address 0x0:
+
+// These modules only change resources stored under the three addresses:
+// LibraAssociation's address: 0xA550C18
+// ValidatorSet address: 0x1D8
+// DiscoverySet address: 0xD15C0
+// ValidatorSet and DiscoverySet are stored under different addresses in order to facilitate cheaper reads of these sets from storage. The keys to these addresses are also owned by the Libra Association entity, therefore the caller on all of the methods is the LibraAssociation's address.
+module LibraSystem2 {
+    use 0x0::LibraAccount;
+    use 0x0::ValidatorConfig2;
+    use 0x0::Vector;
+    use 0x0::Transaction;
+    use 0x0::LibraTimestamp;
+
+    struct ValidatorInfo {
+        addr: address,
+        consensus_voting_power: u64,
+        config: ValidatorConfig2::Config,
+        // To protect against DDoS, the force updates are allowed
+        // only if the time passed since the last force update is larger than a threshold (24hr).
```

Comment:
does move support consts? could we refer to a const?

Reply:
Would be great, I think there are no support for constants in move at the moment... (@sblackshear ?)

Reply:
Not yet,  but soon! @tnowacki is on it